src/transactions/cdp.js

```diff
@@ -626,6 +629,7 @@ export async function addCollateral(accountID, newAlgos) {
   const response = await sendTxn(stxns, "Successfully added " + newAlgos + " ALGOs as collateral.",);
   setLoadingStage(null)
 
+  updateDBWebActions(2, accountID, microNewAlgos, 0, 0, 1000)
```

Comment:
transactionFees should be 2000 here, and microNewAlgos should be negative, right? Since it's from the perspective of the owner wallet?

Reply:
yea, that makes sense